### DISCUSSION TITLE
Application of TRIQS on a Quantum Computer simulator

### DISCUSSION THREAD

**Discussion by TommyHenz** [TRIQS/triqs#869] [Q&A]
Hi everyone, I'm currently working on DFT+DMFT calculations and I've used the framework QE+Wannier90+TRIQS so far. I'd like to ask if you believe there are examples where TRIQS have been used on a Quantum Computer simulator? I want to solve the quantum impurity problem on quantum devices.

Are you aware of that on where can I find worked out examples on how to combine these techniques on a Quantum Computer? 

Sincerely,

**Comment by Wentzell** [comment]
Dear @TommyHenz,

To my knowledge there are no examples of using TRIQS in combination with a Quantum Computer simulator.
Using Quantum Computers for the solution of Quantum Impurity models is however an active Topic of research,
see for example https://arxiv.org/pdf/2108.10780.pdf

**Comment by tayral** [comment]
Hello @TommyHenz 
We are developing an open-source Python library to facilitate the execution of fermionic many-body methods on quantum computers: it is called myQLM-fermion, its documentation is here: https://myqlm.github.io/qat-fermion.html. 
In particular, it helps turning fermionic Hamiltonian into spin (qubit) Hamiltonians (with several choices of transforms like Jordan-Wigner),  write corresponding quantum circuits to run quantum algorithms like Variational Quantum Eigensolver (VQE) or Quantum Phase Estimation (QPE), and finally simulate their execution on quantum machines (or run them directly on a quantum computer).
It should be quite easy to use TRIQS objects in combination with this library. I am happy to give guidance if needed!

**Reply by AQuantumGuy** [reply]
Hi Thomas, I'm writing to inform that I'm quite interested myQLM-fermion library since I'm interested in running my simulations with VQE. As the original poster states, I've also run simulations with DFT+DMFT and now I'm intereted in performing Quantum simulations. Can I contact you to tell you more about my research so you can tell whether myQLM would be ideal?

**Reply by tayral** [reply]
Hello @AQuantumGuy,
sure, you can write to me here or to thomas (dot) ayral (at) atos (dot) net.
